packages/ssr/src/routing/routing.ts

```diff
@@ -93,16 +93,24 @@ export const getRouteDestination = ({
 }) => {
   try {
     const requestUrl = new URL(req.url)
+
+    const { searchParamsWithDefaults, pathParams } = getParameters({
+      route: route.source,
+      req,
+    })
+
     const url = getUrl(
       route.destination,
       // destination formulas should only have access to URL parameters from
       // the route's source definition + global formulas.
       {
         data: {
-          'URL parameters': getDataUrlParameters({
-            route: route.source,
-            req,
-          }),
+          'URL parameters': {
```

Comment:
I guess this should follow your type change?
```suggestion
          'Route parameters': {
```

Reply:
😬